easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2023.12-iimkl-2023b.eb

```diff
@@ -0,0 +1,120 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2023.12'
```

Comment:
FYI, this version is being changed to `2023.11` in https://github.com/easybuilders/easybuild-easyconfigs/pull/24931, to align it with `SciPy-bundle-2023.11-gfbf-2023b.eb`...